docs/webpack/configure-build.md

```diff
@@ -347,28 +347,11 @@ To create the bundle files for production, add the following script to your proj
 
 ## 6. Set environment variables
 
-To deal with environment variables, webpack suggest using the [--env option](https://webpack.js.org/guides/environment-variables/) from it's CLI. While that would work, the environment variables would then only be available to the webpack configuration file:
-
-```js !#6 webpack.build.js
-// @ts-check
-
-import { defineBuildConfig } from "@workleap/webpack-configs";
-import { swcConfig } from "./swc.build.js";
-
-export default (env) => {
-    if (env.DEBUG === "true") {
-        console.log("Configuring webpack in debug mode!");
-    }
-
-    return defineBuildConfig({
-        swcConfig
-    });
-}
-```
+To deal with environment variables, the [webpack](https://webpack.js.org/) documentation suggest using the [--env option](https://webpack.js.org/guides/environment-variables/) from it's CLI. While that would work, by using webpack `--env` CLI option, the environment variables would only be made available to the webpack configuration files (.e.g. `webpack.config.js`) rather than any [Node.js](https://nodejs.org/en) files. Therefore we **do not recommend** using webpack `--env` CLI option.
```

Comment:
```suggestion
To deal with environment variables, the [webpack](https://webpack.js.org/) documentation suggests using the [--env option](https://webpack.js.org/guides/environment-variables/) from its CLI. While that would work, by using webpack `--env` CLI option, the environment variables would only be made available to the webpack configuration files (.e.g. `webpack.config.js`) rather than any [Node.js](https://nodejs.org/en) files. Therefore we **do not recommend** using webpack `--env` CLI option.
```